source/fe/fe_raviart_thomas.cc

```diff
@@ -248,6 +248,173 @@ FE_RaviartThomas<dim>::initialize_support_points(const unsigned int deg)
 }
 
 
+template <int dim>
+void
+FE_RaviartThomas<dim>::initialize_quad_dof_index_permutation_and_sign_change()
+{
+  // For 1D do nothing.
+  //
+  // TODO: For 2D we simply keep the legacy behavior for now. This should be
+  // changed in the future and can be taken care of by similar means as the 3D
+  // case below. The legacy behavior can be found in fe_poly_tensor.cc in the
+  // function internal::FE_PolyTensor::get_dof_sign_change_h_div(...)
+  if (dim < 3)
+    return;
+
+  // TODO: the implementation makes the assumption that all faces have the
+  // same number of dofs
+  AssertDimension(this->n_unique_faces(), 1);
+  const unsigned int face_no = 0;
```

Comment:
Just replace it by zero everywhere below :-)

Reply:
Should I really remove it? It is a small issue that someone probably kept to have not too much work or at least to identify places to change things once other element types enter the game. It is used also in other places in this file. See for example `initialize_restriction()`